packages/test/test-end-to-end-tests/src/test/batching.spec.ts

```diff
@@ -283,7 +283,8 @@ describeFullCompat("Flushing ops", (getTestObjectProvider) => {
                 verifyBatchMetadata(dataObject2BatchMessages);
             });
 
-            it("can send and receive consecutive batches that are flushed on JS turn", async () => {
+            // Disabled due to issue #9546
+            it.skip("can send and receive consecutive batches that are flushed on JS turn", async () => {
```

Comment:
we should scope our disables to only where necessary? is this failing everywhere?

Reply:
read to bug. sounds good